.github/workflows/pr.yml

```diff
@@ -0,0 +1,24 @@
+name: Test
+on:
+  pull_request:
+    branches: [ "master" ]
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [14.x,16.x,18.x]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - run: npm ci
```

Comment:
I've been using it here: https://github.com/bryopsida/onvif/pull/1 to iterate on adding analytics configuration support when I have time, it's been a bit since I've had time to work on that but it's been working there, the test failure is for the case I've been working on and expected, the other cases pass.

Reply:
Thanks! Nice PR!